examples/diffusers/Readme.md

```diff
@@ -0,0 +1,54 @@
+# Running Stable diffusion model using Huggingface diffusers in Torchserve.
+
+### Step 1: Download model
```

Comment:
@jagadeeshi2i It would be great if we could integrate the steps in bash script and keep the steps as for explanations that what it does.